src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2258,6 +2181,7 @@ public boolean isFeedbackSessionFullyCompletedByStudent(
         return true;
     }
 
+    @SuppressWarnings("PMD.UnusedPrivateMethod")
     private boolean isFeedbackSessionFullyCompletedByInstructor(
```

Comment:
@whipermr5 This method is not currently in use, as it was used in `Case PRIVATE:`, but this seems to be a useful method, that we can use later for other purposes (Maybe), which is why I did not remove it. Let me know if you decide otherwise.

Reply:
Don't let "we can use later for other purposes" be a reason against removing dead code, unless that "later" is very immediate. "This class/method/... looks useful, let's save it for a rainy day" is very often the main reason for accumulating dead code.

Reply:
Okay. Removing it. :sweat_smile: